crates/ruff_linter/src/rules/refurb/rules/reimplemented_operator.rs

```diff
@@ -213,11 +213,17 @@ fn subscript_slice_to_string<'a>(expr: &Expr, locator: &Locator<'a>) -> Cow<'a,
     if let Expr::Slice(expr_slice) = expr {
         Cow::Owned(slice_expr_to_slice_call(expr_slice, locator))
     } else if let Expr::Tuple(tuple) = expr {
-        if tuple.parenthesized {
-            Cow::Borrowed(locator.slice(expr))
-        } else {
-            Cow::Owned(format!("({})", locator.slice(tuple)))
-        }
+        let inner = tuple
+            .elts
+            .iter()
+            .map(|expr| match expr {
+                Expr::Slice(expr) => Cow::Owned(slice_expr_to_slice_call(expr, locator)),
+                _ => Cow::Borrowed(locator.slice(expr)),
+            })
+            .collect::<Vec<_>>()
+            .join(", ");
```

Comment:
Nit: `Itertools::join`
```suggestion
            .join(", ");
```